new-nextjs-client/src/styles/reusable-ui-variables.ts

```diff
@@ -27,11 +27,33 @@ const selectOptionBgColorSelectedDark = '#354E80';
   AppToast
 ****************/
 
+// Light theme
+const toastBgSuccessColor = '#38A169';
```

Comment:
change to `toastSuccessBgColor` so that later we can search all background color variables with `bgColor`

Adjust all other variable names below as well.